main.py

```diff
@@ -20,6 +20,25 @@ def one_week_from_now():
     next_week = datetime.today() + timedelta(days=7)
     return next_week.strftime(r"%Y-%m-%d")
 
+def get_next_week_days():
+    result = []
+    
+    if not config["days_of_week"]:
```

Comment:
Please use config.get() for backwards compatibility
```suggestion
    if not config.get("days_of_week", []):
```